I think this will fix it.

Fix compilation error introduced in 484d82b6.

do_unmap_single() was renamed to swiotlb_tbl_unmap_single() in
bfc5501f.  Use the old name unless/until that change is also
backported.

Signed-off-by: David Engel <david@istwok.net>

diff --git a/lib/swiotlb.c b/lib/swiotlb.c
index 7740ee8..0d0df11 100644
--- a/lib/swiotlb.c
+++ b/lib/swiotlb.c
@@ -632,7 +632,7 @@ dma_addr_t swiotlb_map_page(struct device *dev, struct page *page,
 	 * Ensure that the address returned is DMA'ble
 	 */
 	if (!dma_capable(dev, dev_addr, size)) {
-		swiotlb_tbl_unmap_single(dev, map, size, dir);
+		do_unmap_single(dev, map, size, dir);
 		dev_addr = swiotlb_virt_to_bus(dev, io_tlb_overflow_buffer);
 	}
 

